docs/core/diagnostics/specialized-diagnostics-overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Specialized Diagnostics
+description: A guide to more advanced diagnostics support in .NET
+ms.date: 05/19/2023
+---
+# Specialized Diagnostics
+
+If debugging or observability is not sufficient, .NET supports additional diagnostic mechanisms.
+
+## Tracing with Event Source
+
+[Event Source](./eventsource.md)) provides the ability to collect detailed diagnostic information about what is happening inside .NET processes, and includes telemetry information for the Runtime, GC, Libraries and application code.
+
+Event Source data can be collected in-process using the <xref:System.Diagnostics.Tracing.EventListener?displayProperty=nameWithType> API or with external diagostics tools such as [Visual Studio](/visualstudio/profiling), [dotnet-monitor](./dotnet-monitor.md), [dotnet-trace](./dotnet-trace.md), [PerfView](https://github.com/microsoft/perfview) and the [Perfcollect](./trace-perfcollect-lttng.md) scripts. Using the external tools to collect eventsource data in traces is commonly used for perforance analysis.
+
+### Collecting diagnostics in containers
+
+The same diagnostics tools that are used in non-containerized Linux environments can also be used to [collect diagnostics in containers](diagnostics-in-containers.md). There are just a few usage changes needed to make sure the tools work in a Docker container.
+
+### EventPipe
+
+[EventPipe](./eventpipe.md) is a runtime component that can be used to collect tracing data, similar to ETW or LTTng. The goal of EventPipe is to allow .NET developers to easily trace their .NET applications without having to rely on platform-specific OS-native components such as ETW or LTTng.
+
+EventPipe is the mechanism behind many of the diagnostic tools and can be used for consuming events emitted by the runtime as well as custom events written with [EventSource](xref:System.Diagnostics.Tracing.EventSource).
```

Comment:
```suggestion
EventPipe is the mechanism behind many of the diagnostic tools. It can be used for consuming events emitted by the runtime as well as custom events written with [EventSource](xref:System.Diagnostics.Tracing.EventSource).
```